### PR TITLE
Release notes for 2.0.19

### PR DESCRIPTION
**Pull request by derekbekoe**

### CODE REVIEW

**Comment by derekbekoe** [comment]
cc: @sptramer, @JasonRShaver

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[latest/docs-ref-conceptual/release-notes-azure-cli.md](https://github.com/derekbekoe/azure-docs-cli-python/blob/cli-release-2.0.19/latest/docs-ref-conceptual/release-notes-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-latest/release-notes-azure-cli?view=azure-cli-latest&branch=pr-en-us-512) (azure-cli-latest) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C10%5C9%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201710092259492888-512%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=9sGyJOpkvRLBIEPBqwiNageIveNtZmIS1gfX4HZQ%2Fi8%3D&st=2017-10-09T22%3A57%3A18Z&se=2017-11-09T23%3A02%3A18Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: latest/docs-ref-conceptual/release-notes-azure-cli.md [hunk 1]
```diff
@@ -16,6 +16,63 @@ ms.assetid: ce0428f7-0a59-4e72-9237-d907b171af51
...
+
+* Initial release of Batch AI module.
+
+### Keyvault
+
+* Fixed Key Vault authentication issue when using ADFS on Azure Stack. https://github.com/Azure/azure-cli/issues/4448
+
+### Network
+
+* `application-gateway address-pool create`: `--server` argument is not optional to allow creation of empty address pools.
```
Comment (sptramer) [inline latest/docs-ref-conceptual/release-notes-azure-cli.md]: I can't tell what this actually is. Did it fix a bug where creating empty pools was possible, or did it make the `--server` argument optional, or did it change behavior so an empty `--server` value creates an empty pool?

File: latest/docs-ref-conceptual/release-notes-azure-cli.md [hunk 2]
```diff
@@ -16,6 +16,63 @@ ms.assetid: ce0428f7-0a59-4e72-9237-d907b171af51
...
+
+### Sql
+
+* Adding support for SQL Transparent Data Encryption (TDE) and TDE with Bring Your Own Key
+* Added az sql db list-deleted command and az sql db restore --deleted-time parameter, allowing the ability to find and restore deleted databases.
+* Added az sql db op list and az sql db op cancel, allowing the ability to list and cancel in-progress operations on database.
+
+### Storage
+
+* File share snapshot
```
Comment (sptramer) [inline latest/docs-ref-conceptual/release-notes-azure-cli.md]: No context, I need to know whether this was added, removed, changed, etc.

File: latest/docs-ref-conceptual/release-notes-azure-cli.md (hunk 1 above)
Comment (derekbekoe) [reply latest/docs-ref-conceptual/release-notes-azure-cli.md]: It makes the param `--server` optional.
It closes https://github.com/Azure/azure-cli/issues/4533

File: latest/docs-ref-conceptual/release-notes-azure-cli.md (hunk 2 above)
Comment (derekbekoe) [reply latest/docs-ref-conceptual/release-notes-azure-cli.md]: It's a new feature that we now support.
I've modified this bullet point slightly.
https://azure.microsoft.com/en-us/blog/announcing-azure-files-share-snapshots-public-preview/

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[latest/docs-ref-conceptual/release-notes-azure-cli.md](https://github.com/derekbekoe/azure-docs-cli-python/blob/cli-release-2.0.19/latest/docs-ref-conceptual/release-notes-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-latest/release-notes-azure-cli?view=azure-cli-latest&branch=pr-en-us-512) (azure-cli-latest) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C10%5C10%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201710101813516471-512%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=j9ROKzNh891ZWV8B1gOeEYr8TZipiL691I%2BMjPcPn5A%3D&st=2017-10-10T18%3A10%3A59Z&se=2017-11-10T18%3A15%3A59Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[latest/docs-ref-conceptual/release-notes-azure-cli.md](https://github.com/derekbekoe/azure-docs-cli-python/blob/cli-release-2.0.19/latest/docs-ref-conceptual/release-notes-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-latest/release-notes-azure-cli?view=azure-cli-latest&branch=pr-en-us-512) (azure-cli-latest) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C10%5C10%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201710102248515067-512%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=O9UrXCVwhqe1%2FLfQVeMDX1HFI2%2FOj3XxXl%2FBGvTobLw%3D&st=2017-10-10T22%3A46%3A04Z&se=2017-11-10T22%3A51%3A04Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[latest/docs-ref-conceptual/release-notes-azure-cli.md](https://github.com/derekbekoe/azure-docs-cli-python/blob/cli-release-2.0.19/latest/docs-ref-conceptual/release-notes-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/azure-cli-latest/release-notes-azure-cli?view=azure-cli-latest&branch=pr-en-us-512) (azure-cli-latest) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C10%5C10%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201710102257389697-512%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=EywYU8SteAtlmxjtuFXiAX%2F3mnv1vc2HGx0NV0w1ruA%3D&st=2017-10-10T22%3A54%3A41Z&se=2017-11-10T22%3A59%3A41Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**